packages/@webex/internal-plugin-metrics/test/unit/spec/business/business-metrics.ts

```
@@ -181,6 +181,42 @@ describe('internal-plugin-metrics', () => {
+              appType: 'WxCC Desktop',
+              indexHint: 'wxccdesktop',
+              client_timestamp: requestCalls[0].eventPayload.client_timestamp, // This is to bypass time check, which is checked below.
+              value: {
+                bar: "gee",
+                browser: getBrowserName(),
+                browserHeight: window.innerHeight,
+                browserVersion: getBrowserVersion(),
+                browserWidth: window.innerWidth,
+                domain: window.location.hostname,
+                inIframe: false,
+                locale: window.navigator.language,
```

Biome

[error] This property value named locale is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named locale


(lint/suspicious/noDuplicateObjectKeys)